resources/ctia-default.properties

```diff
@@ -4,7 +4,7 @@ ctia.store.sql.db.classname=org.h2.Driver
 ctia.store.sql.db.subprotocol=h2
 ctia.store.sql.db.subname=/tmp/ctia-h2-db;DATABASE_TO_UPPER=false
 ctia.store.sql.db.delimiters=
-ctia.store.es.uri=http://localhost:9200
+ctia.store.es.uri=http://192.168.99.100:9200
 ctia.store.es.indexname=ctia
-ctia.producer.es.uri=http://127.0.0.1:9200
+ctia.producer.es.uri=http://192.168.99.100:9200
```

Comment:
Stephen Sloan <notifications@github.com> writes:

> In resources/ctia-default.properties:
>
>>  ctia.store.es.indexname=ctia
>> -ctia.producer.es.uri=http://127.0.0.1:9200
>> +ctia.producer.es.uri=http://192.168.99.100:9200

Yah, that should not have been hanged in the PR.  The default shoudl
stay localhost.  update your own ctia.properties file, which is
gitignore'ed

>
> Is the IP address change intentional? Maybe you should set the IP in
> your local ctia.properties. If it is intentional, I recommend that you
> make this change in a separate PR.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
>
>

-- 
Craig Brozefsky
Principal Engineer, AMP Threat Grid
Cisco Security Business Group
+1-773-469-8349